Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -501,6 +537,18 @@ private void UpdateRoundFlow(float frameTime)
             {
                 RoundLengthMetric.Inc(frameTime);
             }
+            else if (RunLevel == GameRunLevel.PostRound)
+            {
+                _expandPvsEntities.Clear();
+
+                foreach (var entity in _expandPvsPlayers)
+                {
+                    if (Deleted(entity))
+                        continue;
+
+                    _expandPvsEntities.AddRange(Transform(entity).ChildEntities);
```

Comment:
This should be recursive. i.e., there might be situations where their equipment visuals depends on whether that equipment has a battery or id card inside of it.